tmk_core/common/mousekey.c

```diff
@@ -68,11 +68,11 @@ static uint8_t move_unit(void)
 {
     uint16_t unit;
     if (mousekey_accel & (1<<0)) {
-        unit = (MOUSEKEY_MOVE_DELTA * mk_max_speed)/4;
+        unit = MOUSEKEY_ACCEL0_SPEED((MOUSEKEY_MOVE_DELTA * mk_max_speed));
```

Comment:
Eg, using this may be better:
```suggestion
        unit = (MOUSEKEY_MOVE_DELTA * mk_max_speed)/MOUSEKEY_ACCEL0_SPEED;	
```